litecov.cpp

```diff
@@ -327,6 +329,7 @@ void LiteCov::CollectCoverage(ModuleCovData *data) {
 
   free(buf);
 
+  data->has_remote_coverage = false;

```

Comment:
It's good that you added this line, but it needs to go _after_ `ClearRemoteBuffer(data); ` like this:

```
  ClearRemoteBuffer(data);
  data->has_remote_coverage = false;
```

Otherwise, one of the first things that `ClearRemoteData` does is
`if (!data->has_remote_coverage) return;`
so it never actually does anything.